src/BitStringAddresses/bitstring.jl

```diff
@@ -1,13 +1,19 @@
 """
     num_chunks(::Val{B})
 
-Determine the number of chunks needed to store `B` bits.
+Determine the number and type of chunks needed to store `B` bits.
 """
 function num_chunks(::Val{B}) where {B}
     return if B ≤ 0
```

Comment:
Maybe off topic, but why is there a `return` in this line? Would
```suggestion
    if B ≤ 0
```
work the same?

Reply:
`return` at the end doesn't do anything and is just a stylistic thing. Maybe putting it before the if is a bit weird, I'll move it.